arviz/plots/ecdfplot.py

```diff
@@ -0,0 +1,107 @@
+"""Plot ecdf and ecdf-difference plot with confidence bands."""
+import numpy as np
+
+from arviz.rcparams import rcParams
+from arviz.plots.plot_utils import get_plotting_function
+
+
+def plot_ecdf(
+    values,
+    values2=None,
+    distribution=None,
+    difference=False,
+    pit=False,
+    confidence_bands=False,
+    granularity=100,
+    num_trials=500,
+    alpha=0.95,
+    figsize=None,
+    ecdf_fill=True,
+    ax=None,
+    show=None,
+    backend=None,
+    backend_kwargs=None,
+    **kwargs
+):
+    """Plot ECDF and ECDF-Difference Plot with Confidence bands.
+
+    Parameters
+    ----------
+    values : array-like
+        Values to plot from an unknown continuous distribution
+    values2 : array-like, optional
+        Values to compare to the original sample
+    distribution : function, optional
+        Cumulative distribution function of the distribution we will compare our sample to
+    pit : bool, optional
+        If True plots the ECDF or ECDF-diff of PIT of sample
+    confidence_bands : bool, optional
+        If True plots the simultaneous confidence bands with 1 - alpha confidence level
+    granularity : int, optional
+        This denotes the granularity size of out plot
+    num_trails : int, optional
+        The number of trials to carry while finding gamma
```

Comment:
It seems like it would be worthwhile to have an argument like `simultaneous=True` or `pointwise=False` that computes the pointwise (ie. marginal) confidence bands.